pkg/graveler/retention/expired_commits.go

```diff
@@ -39,9 +39,9 @@ func GetGarbageCollectionCommits(ctx context.Context, startingPointIterator *GCS
 	// All further commits in the ancestry are added to the expired set. The iteration stops upon reaching a commit which exists in the previouslyExpired set, or the DAG root.
 	processed := make(map[graveler.CommitID]time.Time)
 	// Mapping between previously expired commits to their direct children.
-	previouslyExpiredMap := make(map[graveler.CommitID]map[graveler.CommitID]struct{})
+	prevExpiredCommitsToChildrenMap := make(map[graveler.CommitID]map[graveler.CommitID]struct{})
```

Comment:
Best var name ever!  Makes the "`// Mapping...`" comment that I asked for almost useless :-)